PWGHF/TableProducer/refitPvDummy.cxx

```diff
@@ -70,7 +77,27 @@ struct HfRefitPvDummy {
       // fill version 001 of candidate table, containing also the collision ID
       rowProng3WithCollId(collision.globalIndex(), rowTrackIndexProng3.prong0Id(), rowTrackIndexProng3.prong1Id(), rowTrackIndexProng3.prong2Id(), rowTrackIndexProng3.hfflag());
     }
-  } // end of process function
+  } // end of process function for 3 prong candidates
+
+  PROCESS_SWITCH(HfRefitPvDummy, process3Prongs, "Produce tables for 3-prong candidates", true);
+
+  // process function for cascade candidates
+  void processCascades(aod::Collisions const& collisions,
+                       aod::HfCascades_000 const& rowsTrackIndexCascade,
+                       aod::Tracks const&)
+  {
+    // dummy tables for cascade candidates
+    for (const auto& rowTrackIndexCascade : rowsTrackIndexCascade) {
+
+      // original PV information
+      auto track0 = rowTrackIndexCascade.prong0_as<aod::Tracks>();
```

Comment:
The casting should not be needed here, if the native index column type is already `Tracks`, no?

Reply:
You are right indeed, I only kept as it was before, but now I removed it! 👍

Reply:
Thanks